frontend/trip-planner/src/Components/EmailAdd.js

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { ReactMultiEmail, isEmail } from 'react-multi-email';
+import 'react-multi-email/dist/style.css';
+
+function Email(props) {
+  //const [emails, setEmails] = useState([]);
+  const [focused, setFocused] = useState(false);
+
+  return (
+    <form className='emails'>
```

Comment:
I'm not sure you need this `<form ..>`. The parent component has a `<form ..>` already.